src/main/java/net/dries007/tfc/compat/jei/TFCJEIPlugin.java

```diff
@@ -337,5 +338,11 @@ public void register(IModRegistry registry)
         registry.addAdvancedGuiHandlers(new TFCInventoryGuiHandler<>(GuiCalendar.class));
         registry.addAdvancedGuiHandlers(new TFCInventoryGuiHandler<>(GuiNutrition.class));
         registry.addAdvancedGuiHandlers(new TFCInventoryGuiHandler<>(GuiSkills.class));
+
+        //Add JEI descriptions for basic mechanics
+
+        registry.addDescription(new ItemStack(BlocksTFC.PIT_KILN,1), new TextComponentTranslation(TerraFirmaCraft.MOD_ID + ".jei.description.pitkiln").getFormattedText());
+        registry.addDescription(new ItemStack(BlocksTFC.PLACED_ITEM,1), new TextComponentTranslation(TerraFirmaCraft.MOD_ID + ".jei.description.placeditem").getFormattedText());
```

Comment:
From JEI API:
```
@deprecated since JEI 4.5.0. Use {@link #addIngredientInfo(List, IIngredientType, String...)}
```

Please change this to not use the deprecated API and use `addIngredientInfo` instead with `VanillaTypes.ITEM`.